lib/spack/spack/ci.py

```diff
@@ -315,18 +315,16 @@ def print_staging_summary(spec_labels, dependencies, stages):
         return
 
     tty.msg('  Staging summary:')
```

Comment:
If @scottwittenburg argues that we need to list all the jobs, including those does not need a rebuild, then the minimum we could do is add some sort of legend here.
```
tty.msg('  Staging summary: ([x] -> needs rebuild)')
```

Note that this is no longer true with the current PR proposal, which is to only list specs that will be rebuilt.